iaam/i40fd/README.md

```diff
@@ -0,0 +1,11 @@
+# Industry 4.0 Field Device Ontology
+
+This ontology provides concepts that allow interoperability at the application layer of field devices (e.g., IO-Link, PROFINET, CANOpen)
+and allows a generalization based on device profiles, units of measurement and communication properties.
```

Comment:
```suggestion
and allows a generalization based on device profiles, units of measurement, and communication properties.
```

Reply:
Thanks for the feedback. I have added the comma.